### DISCUSSION TITLE
Todo Mode, but for Notes apps as well

### DISCUSSION THREAD

**Discussion by MishaScholte** [rxhanson/Rectangle#529] [Ideas]
I absolutely love Todo mode. I already had my Todo app on the 6th quarter, but always had a problem with resizing other screens. This totally fixed it!

However, to add: I also keep my Notes app on hand, and I use the first quarter for that. I was wondering if it would be a good idea to expand this feature to also be able to pin an app on the left side

**Comment by rxhanson** [comment]
Thanks for the feedback. I like this idea. I don't have the bandwidth to add this in at the moment, but would consider a pull request.

**Comment by LeonardoGentile** [comment]
Instead of having a todo mode and a Note mode, why not simply adding three custom modes and let the user decide which app to use on the side on the screen for mode 1, 2 and 3?

Technically is the same, isn't it?

**Reply by rxhanson** [reply]
This sort of starts down a rabbit hole of auto-tiling windows, which is something I'm currently working on the foundation of in Rectangle Pro (increased flexibility, but increased complexity). If anyone in the community wants to expand the functionality of Todo mode, I don't mind merging in updates to it.
